src/routes/+layout.svelte

```diff
@@ -128,11 +128,11 @@
 		unstyled: true,
 		classes: {
 			toast:
-				'shadow-xl px-4 py-3 flex items-center gap-x-3 max-w-full w-full rounded mx-auto text-xs mx-0',
+				'shadow-xl px-4 py-3 flex items-center gap-x-3 max-w-full w-full rounded mx-auto text-xs mx-0 !outline-none',
 			loading: 'bg-shade-0',
 			error: 'text-red-50 bg-red-700',
 			success: 'text-emerald-50 bg-emerald-700',
-			warning: 'text-yellow-50 bg-yellow-700',
+			warning: 'text-yellow-50 bg-yellow-700 !bg-yellow-700',
```

Comment:
> I guess the problem is with the playwright test itself, because when running a local server and manually executes the test in the browser (I am using Chrome 138.0.7204.157 (Official Build) (64-bit) under Pop OS!) the toast styles are fine.

Oh, great catch! I think you are right, I just re-ran the test but took a screenshot instead of using the Trace Viewer and the colors are correct!

We can undo the hacks 👍